Sources/PackageLoading/PackageDescription4Loader.swift

```diff
@@ -141,6 +141,8 @@ enum ManifestJSONParser {
         let providers = try? json
             .getArray("providers")
             .map(SystemPackageProviderDescription.init(v4:))
+        
+        let capability = try? TargetDescription.ExtensionCapability(v4: json.getJSON("extensionCapability"))
```

Comment:
would we loose the error information?

Reply:
Yes, that's a good point.  I did the same as the previous code in this method (see the line just above) but we can improve this.  It's true though that this is an internal contract between two parts of SwiftPM (which doesn't mean we shouldn't have error checking here) but that seems less important than error checking for things the user controls.